### PR TITLE
Tdd/park info

### LINKED ISSUE #18: Testing for About Component

### LINKED ISSUE #25: Testing for ParkInfo Component

### LINKED ISSUE #27: Testing for SavedCard Component

### LINKED ISSUE #45: Testing for Contact Component

### LINKED ISSUE #46: Testing for Location Component

### PR DESCRIPTION
**Pull request by saraho1123**
## Is this a fix or a feature?
feature/testing

## What is the change?
Unit tests the following components:
* `ParkInfo`
* `About`
* `Location`
* `Contact`
*  `SavedCards`

## Where should the reviewer start?
Review the test files for each of the above components.

## How should this be tested?
1. See if there are sufficients tests for each component
2. run tests to confirm they are passing with:  `npm test src/<folder name>/<component name>.test.js`

## Relevant Tickets
* Closes #25
* Closes #18
* Closes #27
* Closes #45
* Closes #46

@saraho1123
@jaypeasee
@baileydunning

### CODE REVIEW

File: src/SavedCard/SavedCard.test.js [hunk 1]
```diff
@@ -1 +1,105 @@
...
+
+import { screen, render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import SavedCard from './SavedCard'
+import { samplePark } from '../samplePark'
+import { Router } from 'react-router-dom'
+import { createMemoryHistory } from 'history'
+import '@testing-library/jest-dom'
+
+describe('SavedCard', () => {
```
Comment (baileydunning) [inline src/SavedCard/SavedCard.test.js:8]: Nice organization with these

**Review by baileydunning — APPROVED** [review]
Great job on these unit tests, Sarah!

File: src/ParkInfo/ParkInfo.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    parkCode: 'dena',
+    visited: false,
+    bucketList: false
+  }
+
+  beforeEach(() => {
+    render(
+      <Router history={history}>
+        <ParkInfo
+          currentPark={samplePark}
```
Comment (jaypeasee) [inline src/ParkInfo/ParkInfo.test.js:28]: We should move this data into a file of it's own and import it.

File: src/SavedCard/SavedCard.test.js [hunk 3]
```diff
@@ -1 +1,103 @@
...
+      state: 'AK',
+      image: 'https://media.nationalgeographic.org/assets/photos/172/369/81ad2d66-4da3-43c1-b5ff-5d30ace2ca2e.jpg',
+      parkCode: 'dena',
+      visited: true,
+      bucketList: false
+    }
+    const history = createMemoryHistory()
+
+    beforeEach(() => {
+      history.location.pathname = '/user/visited'
```
Comment (jaypeasee) [inline src/SavedCard/SavedCard.test.js:26]: Nice job manipulating the history object here.

**Review by jaypeasee — APPROVED** [review]
Nice Work! Unit Tests DONE. Only conflict I see is in the ESLint Cache.

**Pull request closed without merging**